pkgs/development/compilers/gerbil/smug-gerbil.nix

```diff
@@ -1,21 +1,16 @@
-{ pkgs, lib, fetchFromGitHub, gerbil-unstable, gerbil-support, gambit-support }:
-
-gerbil-support.gerbilPackage {
+{ lib, fetchFromGitHub, ... }:
+{
```

Comment:
```suggestion
{ lib, fetchFromGitHub }:

{
```

Reply:
Nah, the `...` is necessary, because I always call with plenty of arguments from `prePackage-defaults`. Not sure how best to do things otherwise.